core/src/main/java/io/confluent/rest/Application.java

```diff
@@ -118,18 +118,26 @@ public Application(T config, String path) {
   }
 
   public Application(T config, String path, String listenerName) {
+    this(config, path, listenerName, null);
+  }
+
+  @VisibleForTesting
+  Application(T config, String path, String listenerName, CustomRequestLog customRequestLog) {
```

Comment:
Create this package private constructor so that we are able to mock CustomRequestLog protected final field.